examples/subscribing_to_stream.rs

```diff
@@ -0,0 +1,112 @@
+use eventstore::{
+    Client, Credentials, EventData, ExpectedVersion, Origin, Position, ReadResult, SubEvent,
+    SubscribeToAllOptions, SubscribeToStreamOptions, SubscriptionFilter,
+};
+use futures::TryStreamExt;
+use serde::{Deserialize, Serialize};
+use std::error::Error;
+use uuid::Uuid;
+
+type Result<A> = std::result::Result<A, Box<dyn Error>>;
+
+pub async fn subscribe_to_stream(client: &Client) -> Result<()> {
+    // region subscribe-to-stream
+    client
+        .subscribe_to_stream("some-stream", &Default::default())
+        .await?;
+    // endregion subscribe-to-stream
+
+    // region subscribe-to-stream-from-position
+    let options = SubscribeToStreamOptions::default().origin(Origin::Point(20));
```

Comment:
**CONCERN:** Similar comment as above about the naming. So it should be rather `StreamPosition` than `Origin`. We could also add explicit method `fromRevision` as it's in Java and NodeJS clients.

Reply:
Similar to https://github.com/EventStore/EventStoreDB-Client-Rust/pull/49#discussion_r573642574

Reply:
done